### DISCUSSION TITLE
AS1000_VNAV_TOGGLE

### DISCUSSION THREAD

**Discussion by Noircogi** [nguyenquyhy/Flight-Tracker-StreamDeck#343] [Q&A] [answered]
I'm trying to get the VNV button for the G1000NXi working.  The toggle value above is listed in the release notes for it, but I'm not able to get it to work when I use it as the "Toggle Event" in a Generic Toggle Button.

Is that something which should just work, or is additional support required in the Flight-Tracker code?

It would also be great to have the proper feedback value, if that's possible

Thanks,
Scott

**Comment by Noircogi** [comment, marked as answer]
I got the toggle working using "Mobiflight.AS1000_VNAV_TOGGLE"
I'll do more research in the Mobiflight area to figure out the feedback values.

**Reply by Noircogi** [reply]
From the debugger, the value I want to use for feedback is XMLVAR_VNAVButtonValue
How do I make that work as a Feedback Value in Flight-Tracker?

**Reply by nguyenquyhy** [reply]
XMLVAR_VNAVButtonValue is an LVar, i.e. it is not exposed via SimConnect, so the current plugin won't be able to read the value.
